src/Winium.Desktop.Driver/CommandExecutors/SetOrientationExecutor.cs

```diff
@@ -0,0 +1,50 @@
+namespace Winium.Desktop.Driver.CommandExecutors
+{
+    #region using
+
+    using System;
+
+    using Winium.StoreApps.Common;
+    using Winium.Cruciatus.Core;
+
+    #endregion using
+
+    internal class SetOrientationExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        protected override string DoImpl()
+        {
+            if (!this.ExecutedCommand.Parameters.ContainsKey("orientation"))
+            {
+                // TODO: in the future '400 : invalid argument' will be used
+                return this.JsonResponse(ResponseStatus.UnknownError, "WRONG PARAMETERS");
+            }
+
+            var orientation = (DisplayOrientation)Enum.Parse(
+                typeof(DisplayOrientation), 
+                this.ExecutedCommand.Parameters["orientation"].ToString());
+
+            var result = RotationManager.SetOrientation(orientation);
+
+            switch (result)
+            {
+                case 0:
+                    return this.JsonResponse();
```

Comment:
Turns out there was a miscommunication. Gleb wrongly understood docs, this command should not return orientation. So this is all good in your code.

But there is no `UnableToSetDisplayOrientation = 35` response code in protocol, so could you please replace it with `UnknownError`. If we will send valid response status, then there is no need to log error message, as it will get correctly parsed and displayed on client side. Sending custom response status might break client bindings.